proto/leaderboard.proto

```diff
@@ -94,18 +94,26 @@ message _RankRange {
   uint64 end_exclusive = 2 [jstype = JS_STRING];
 }
 
+message _Unbounded { }
+
 // Query APIs using ScoreRange may match more than the limit of 8192 elements. These apis will
 // include an offset and limit parameter pair, which can be used to page through score ranges
 // matching many elements.
 //
 // ScoreRange models half-open ranges: 0..4 refers to scores 0, 1.1234, 2.5 and 3.999.
 message _ScoreRange {
-  // IEEE 754 single precision 32 bit floating point number.
-  // Momento does not support NaN or Inf in leaderboards.
-  float start_inclusive = 1;
-  // IEEE 754 single precision 32 bit floating point number.
-  // Momento does not support NaN or Inf in leaderboards.
-  float end_exclusive = 2;
+  oneof min {
+    _Unbounded unbounded_min = 1;
+    // IEEE 754 single precision 32 bit floating point number.
+    // Momento does not support NaN or Inf in leaderboards.
+    float min_score = 2;
```

Comment:
Saw this after it was merged - could we rename `min_score` to `inclusive_min` and `max_score` to `exclusive_max`?

Reply:
sure.